src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  const rules = sourceString
+    .split(';')
+    .map((r) => r.trim())
+    .filter((r) => r !== '');
+
+  rules.forEach((rule) => {
+    const [prop, value] = rule.split(':');
```

Comment:
The current implementation of splitting by ':' assumes that there will only be one colon per rule. If the style string's property values contain colons, this will not work correctly. However, according to the additional prompt, handling multiple colons is not required, but it is okay if implemented. Consider whether you want to handle such cases.